projects/react-boilerplate-ts/app/components/TestComponent/ColorButton.story.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import ColorButton from "./ColorButton";
+
+storiesOf("Components", module).add(
```

Comment:
Seem to be mixing single and double quotes. Elsewhere I think we're just doing single quotes.

Reply:
Good spot. Prettier should be correcting this for us. Will have a look at that. Maybe the prettier config file should be in the root of the mono repo?

Reply:
Yea good point, we could pull that out of boilerplate